pkg/services/ngalert/store/alertmanager_test.go

```diff
@@ -185,7 +185,7 @@ func TestIntegrationAlertManagerConfigCleanup(t *testing.T) {
 		SQLStore: sqlStore,
 		Logger:   log.NewNopLogger(),
 	}
-	t.Run("when calling the cleanup with less records than the limit all recrods should stay", func(t *testing.T) {
+	t.Run("when calling the cleanup with fewer records than the limit all recrods should stay", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("when calling the cleanup with fewer records than the limit all records should stay", func(t *testing.T) {
```
If we're already on the topic of grammar, there's this too :laughing: